docs/installation/migrate.rst

```diff
@@ -16,77 +16,130 @@ You can set up continuous migration of |sr| to maintain a hybrid deployment ("ex
 cloud") or "lift and shift" all to |ccloud| using a one-time migration.
 
 Continuous Migration
---------------------
+~~~~~~~~~~~~~~~~~~~~
 
 For continuous migration, you can use your self-managed |sr| as a primary and
 |sr-ccloud| as a secondary. New schemas will be registered directly to the
 self-managed |sr|, and |crep| will continuously copy schemas from it to
 |sr-ccloud|, which is set to IMPORT mode.
 
-**TBD diagram showing continuous migration**
-
 One-time Migration
-------------------
+~~~~~~~~~~~~~~~~~~
 
 Choose a one-time migration to move all data to a fully-managed |ccloud|
 service. In this case, you migrate your existing self-managed |sr| to
 |sr-ccloud| as a primary. All new schemas are registered to |sr-ccloud|. In the
 scenario, there is no migration from |sr-ccloud| back to the self-managed |sr|.
 
-**TBD diagram showing one-time migration**
+Prerequisites
+-------------
 
+The Run Book describes how to perform a |sr| migration applicable to any type of
```

Comment:
The positioning of this page is a bit conflicting.  The following is my interpretation but feel free to adjust as you see fit:

* `runbook`: to me this implies that this page is going to be applicable to a customer's production on-prem deployment.  So the local config is to a production Kafka cluster and SR instance. 
* `confluent` CLI: this would be for a customer's sandbox environment on a localhost.

Consider choosing one or the other: is this page meant to guide the reader through a production migration (in which case don't use Confluent CLI) or it's meant more as a test, in which case don't use the term `runbook`.